docs/arcaflow/workflows/schemas.md

```diff
@@ -0,0 +1,61 @@
+# Workflow Schemas
+
+## Schema Names
+
+### Non-Object Names
+
+Schemas that are not composed within an ObjectSchema do not have an `Object ID`. They use a stringified version of their `TypeID` for their schema name.
+
+| schema        |     type id                 |
+|---------------|-----------------------------|
+| IntEnum       | enum_string                      |
+| StringEnum    | enum_integer                      |
+| String        | string                      |
+| Pattern       | pattern                      |
+| Int           | integer                     |

```

Comment:
Are you sure that `integer` is correct, or should that be `int`?  (The other references, like in `bindConstants()` use `int`.)

Reply:
I don't know what you're referencing

Reply:
I think this pair is probably right as is; I've [flagged](https://github.com/arcalot/arcalot.github.io/pull/81#discussion_r1607530108) two _other_ pairs which might be wrong.